Cargo.toml

```diff
@@ -49,9 +49,12 @@ bb8 = "0.8"
 bb8-postgres = "0.8"
 rusqlite = { version = "0.31.0", features = ["bundled"] }
 tokio-rusqlite = "0.5.1"
-mysql_async = {version = "0.34.1", features = ["native-tls-tls", "chrono"]}
+mysql_async = { version = "0.34.1", features = ["native-tls-tls", "chrono"] }
 tokio-postgres = { version = "0.7.10", features = ["with-chrono-0_4"] }
-clickhouse-rs = {version = "1.1.0-alpha.1", features = ["tokio_io", "tls"]}
+clickhouse-rs = { git = "https://github.com/gloomweaver/clickhouse-rs.git", branch = "async-await", features = [
```

Comment:
Let's avoid pointing to a personal fork.

Reply:
Already update to point to spiceai